src/main/scala/org/hammerlab/guacamole/Serialization.scala

```diff
@@ -48,5 +99,71 @@ class GuacamoleKryoRegistrator extends ADAMKryoRegistrator {
     kryo.register(classOf[Allele], new AlleleSerializer)
     kryo.register(classOf[Genotype], new GenotypeSerializer)
     kryo.register(classOf[TaskPosition])
+
+    kryo.register(classOf[Array[LociSet]])
+    kryo.register(classOf[Map[_, _]])
+    kryo.register(Map.empty.getClass)
+    kryo.register(scala.math.Numeric.LongIsIntegral.getClass)
+    kryo.register(classOf[SimpleRange])
+    kryo.register(classOf[Array[SimpleRange]])
+
+    kryo.register(classOf[Vector[_]])
+    kryo.register(classOf[Array[Vector[_]]])
+    kryo.register(classOf[scala.collection.mutable.WrappedArray.ofLong])
+    kryo.register(classOf[scala.collection.mutable.WrappedArray.ofByte])
+    kryo.register(classOf[scala.collection.mutable.WrappedArray.ofChar])
+    kryo.register(classOf[Array[Char]])
+
+    // Tuple2[Long, Any], afaict?
+    // "J" == Long (obviously). https://github.com/twitter/chill/blob/6d03f6976f33f6e2e16b8e254fead1625720c281/chill-scala/src/main/scala/com/twitter/chill/TupleSerializers.scala#L861
+    kryo.register(Class.forName("scala.Tuple2$mcJZ$sp"))
+    kryo.register(Class.forName("scala.Tuple2$mcIZ$sp"))
+
+    kryo.register(classOf[Array[String]])
+    kryo.register(classOf[Array[Int]])
+
+    // https://mail-archives.apache.org/mod_mbox/spark-user/201504.mbox/%3CCAC95X6JgXQ3neXF6otj6a+F_MwJ9jbj9P-Ssw3Oqkf518_eT1w@mail.gmail.com%3E
+    kryo.register(Class.forName("scala.reflect.ClassTag$$anon$1"))
+    kryo.register(classOf[java.lang.Class[_]])
+
+    kryo.register(classOf[scala.collection.mutable.WrappedArray.ofRef[_]])
+    kryo.register(classOf[Array[Array[Byte]]])
+
+    kryo.register(classOf[Array[AlignmentRecord]])
+    kryo.register(classOf[ReferenceRegion])
+    kryo.register(classOf[Array[ReferenceRegion]])
+
+    kryo.register(classOf[org.bdgenomics.formats.avro.Strand])
+    kryo.register(classOf[org.bdgenomics.adam.models.MultiContigNonoverlappingRegions])
+    kryo.register(classOf[org.bdgenomics.adam.models.NonoverlappingRegions])
+
+    // Added to Spark in 1.6.0; needed here for Spark < 1.6.0.
+    kryo.register(classOf[Array[Tuple1[Any]]])
```

Comment:
Possibly; I guess there's a question about whether having our POM depend on Spark 1.6.1 implies we don't consider earlier versions of Spark to be supported.

I'm fine saying that and removing these.